metriq_gym/dispatch_clops.py

```diff
@@ -18,10 +18,10 @@
 def main():
     args = parse_arguments()
 
-    if args.token:
-        QiskitRuntimeService.save_account(channel="ibm_quantum", token=args.token, set_as_default=True, overwrite=True)
+    if args.token and args.instance:
+        QiskitRuntimeService.save_account(channel="ibm_quantum", token=args.token, instance=args.instance, set_as_default=True, overwrite=True)
 
-    logging.info(f"Dispatching CLOPS job with n={args.num_qubits}, shots={args.shots}, trials={args.trials}, backend={args.backend}, confidence_level={args.confidence_level}, jobs_file={args.jobs_file}")
+    logging.info(f"Dispatching CLOPS job with n={args.num_qubits}, shots={args.shots}, backend={args.backend}, jobs_file={args.jobs_file}")
```

Comment:
Is there a reason that `trials` and `confidence_level` are removed from the logging statement?

Reply:
Because we don't pass them as parameters of the job when we dispatch it.

Reply:
Yeah, that makes sense to me. Marking this as resolved.